lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -20,6 +20,52 @@ class CancelOutStatus {
 // e.g. (2x^2 * 5) / 2x^2 => 5 / 1
 // Returns a Node.Status object
 function cancelLikeTerms(node) {
+  // Cancel out terms if the term being multiplied
+  // is the same as the denominator
+  if (Node.Type.isOperator(node) && node.op === '*') {
+    let newNode = clone(node);
+    let leftNode = newNode.args[0];
+    let rightNode = newNode.args[1];
+    if (Node.Type.isParenthesis(leftNode)) {
+      leftNode = leftNode.content;
+    }
+    // leftNode is a fraction. Check if the denominator is the
+    // same as the rightNode being multiplied
+    if (Node.Type.isOperator(leftNode) && leftNode.op === '/') {
+      if (leftNode.args[1].equals(rightNode)) {
+        newNode = leftNode.args[0];
+        return Node.Status.nodeChanged(
+          ChangeTypes.CANCEL_TERMS, node, newNode);
+      }
+    }
+    // leftNode is the sum of multiple terms. Check if all terms
+    // have the same denominator and matches the rightNode
+    // being multiplied
+    if (Node.Type.isOperator(leftNode) && leftNode.op === '+') {
+      let allMatchingDenominators = true;
+      for (let i = 0; i < leftNode.args.length; i++) {
```

Comment:
you could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every here, I think